content/docs/reference/config/helm-customization/index.md

```diff
@@ -0,0 +1,283 @@
+---
+title: Customise the Helm Installation
+description: Customise the Istio installation with the included Helm chart.
+weight: 30
+keywords: [kubernetes,helm]
+aliases:
+    - /docs/reference/config/helm/index.html
```

Comment:
Only add aliases when moving pages. This is a new page, so it shouldn't have an alias.